stubs/zstd/zstd.pyi

```diff
@@ -0,0 +1,13 @@
+class Error(Exception): ...
+
+def ZSTD_compress(data: str | bytes, level: int | None, threads: int | None) -> str | bytes: ...
```

Comment:
If I remember my C argument codes correctly, the first arg should be `_typeshed.ReadableBuffer` (https://github.com/sergey-dryabzhinsky/python-zstd/blob/master/src/python-zstd.c#L68 - I think y# is buffers). Also, the return type appears to be always `bytes`.

(We don't care about Python 2 in typeshed.)

Reply:
I just tested and you seem to be right, I change it

Reply:
Thanks! I haven't looked at the other functions but I suspect the same things apply elsewhere. A few functions appear to be aliases for this one.

Reply:
Yep there are a lot of aliases, namely

- **ZSTD_compress**
  - compress
  - dumps
- **ZSTD_uncompress**
  - decompress
  - uncompress
  - loads
- **ZSTD_version**
  - version